src/WorldMap.h

```diff
@@ -1,27 +1,96 @@
 #pragma once
 
-#include "TileLayer.h"
-#include "TileSheetDetails.h"
-#include <vector>
+#include <unordered_map>
 #include <SFML\Graphics.hpp>
 #include <string>
 
+class TiXmlElement;
+struct LevelDetails;
```

Comment:
minor nitpick - we should probably use only either struct or class keyword - not both